csaf_files/IT/white/2024/va-24-201-01.json

```diff
@@ -229,8 +229,9 @@
       "acknowledgments": [
         {
           "names": [
-            "Matthew Galligan, CISA Rapid Action Force"
-          ]
+            "Matthew Galligan"
+          ],
+          "organization": "CISA Rapid Action Force",
```

Comment:
https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#311-acknowledgments-type

Properties for acknowledgements are names, organization, summary, and urls. Moved organization from names property to organization property.

Reply:
I agree with the above change.